todoapp/users/models.py

```
@@ -1,17 +1,32 @@
-
+    def create_user(self, email, password ,  **extra_fields):
+        """
+        Create and save a user with the given email, and password.
+        """
+        email = self.normalize_email(email)
+        user = self.model(email=email, **extra_fields)
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+    
+    def create_superuser(self, email , password, **extra_fields):
```

Ruff (0.12.2)

Missing return type annotation for public function `create_superuser`

(ANN201)